packages/php-wasm/universal/src/lib/base-php.ts

```diff
@@ -616,6 +616,39 @@ export abstract class BasePHP implements IsomorphicLocalPHP {
 		this[__private__dont__use].FS.rename(fromPath, toPath);
 	}
 
+	/** @inheritDoc */
+	@rethrowFileSystemError('Could not copy "{path}"')
+	cp(fromPath: string, toPath: string) {
```

Comment:
Let's add an options arg with a single `recursive: bool` option to ensure the developers express an explicit intention to copy a directory

Reply:
https://github.com/WordPress/wordpress-playground/pull/846/files#diff-b33e1e6834fb94ac2cb5338e3ae17c60abf7ac88cb057189a1915531e55f7243R621

Reply:
It seems like you added a boolean arg. Let's use an options arg instead, like I mentioned in that previous comment. It would have just a single option, like `recursive: bool`. So you would call it like `cp(fromPath, toPath, { recursive: true })`. This makes the calls clear, as you don't have to think about what that `true` value means, and it also makes the API easy to extend in the future.

Reply:
@adamziel Ah, I misunderstood. Done.
https://github.com/WordPress/wordpress-playground/blob/0a3ef1eb1a99569921e8238c43314fea8704a174/packages/php-wasm/universal/src/lib/base-php.ts#L622

https://github.com/WordPress/wordpress-playground/blob/0a3ef1eb1a99569921e8238c43314fea8704a174/packages/php-wasm/universal/src/lib/universal-php.ts#L554-L560

Reply:
Thank you! Note that `cp()` signature makes the `options` arg mandatory – let's make it optional like it is in `rmdir`